### PR TITLE
Update Getting Started notebook to use init

### PR DESCRIPTION
**Pull request by naddeoa**
Just a small sample of how the docs/ux will look after the update to include init(). We can use this to make tweaks or back compat fixes before switching more things over.

Gaps:
- There isn't an easy way to make segmented classification/regression metrics work because those don't use why.log under the hood, so I'll have to make a custom integration for them at some point. Just left those examples the same (no init).

### CODE REVIEW

File: python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb [hunk 1]
```diff
@@ -307,101 +309,30 @@
...
-    "writer = WhyLabsWriter().option(reference_profile_name=\"my_reference_profile\")\n",
-    "writer.write(file=profile.view())"
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "> Another way of setting `reference_profile_name` is to define the environment variable `WHYLABS_REFERENCE_PROFILE_NAME`\n",
```
Comment (naddeoa) [inline python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb:363]: @jamie256 Is this true?? I can't find any reference to that name/feature.

Comment (jamie256) [reply python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb:363]: https://github.com/search?q=repo%3Awhylabs%2Fwhylogs+get_whylabs_refernce_profile_name&type=code

The environment variable reference profile name functionality still exists.

**Review by murilommen — DISMISSED** [review]
apart from the pre-commit errors and updating the branch with the latest changes, this looks like what we need for the examples with `why.init`

File: python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb (hunk 1 above)
Comment (naddeoa) [reply python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb:363]: lol I think I added that for feature parity, but I had no idea if it existed before. It's fine I guess, it's there now.

File: python/examples/integrations/writers/Getting_Started_with_WhyLabsV1.ipynb
Comment (FelipeAdachi) [inline python/examples/integrations/writers/Getting_Started_with_WhyLabsV1.ipynb:1]: - Should we add `reinit=True` to init just to make sure? I ask because I'm testing this, and I got old orgs and model ids that are unrelated to the current model I was planning to upload to, and that confused me.

- On a related note, `reinit=True` is not working for me. Even with reinit, it's automatically picking the old org and model ids that I don't want to be picked.

File: python/whylogs/api/whylabs/session/prompts.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import sys
 from typing import List, Optional
+import getpass
```
Comment (jamie256) [inline python/whylogs/api/whylabs/session/prompts.py]: isort?

can you run this locally:
`make pre-commit`

File: python/examples/integrations/writers/Getting_Started_with_WhyLabsV1.ipynb
Comment (FelipeAdachi) [reply python/examples/integrations/writers/Getting_Started_with_WhyLabsV1.ipynb:1]: It looks like reinit doesn't work as I was thinking - I thoush reinit would reprompt me the variables, but instead you need to set the env vars yourself and then call reinit to update the config.

Maybe this is fine, but it's not clear that this is the case. Maybe it's just a matter of clearly explaining how this works? like, on the comments, stating that "If you want to change the variables in your config file, set them with os....etc and then call why.init(reinit=True)

I think that would give enough info to the user that falls in this scenario as I did.

Comment (naddeoa) [reply python/examples/integrations/writers/Getting_Started_with_WhyLabsV1.ipynb:1]: Rather than clutter the profiling code here with init explanations, we could give a very brief description and then link to the more complete doc that explains these nuances?

**Review by FelipeAdachi — APPROVED** [review]
LGTM!
